code/src/index.css

```diff
@@ -5,9 +5,162 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+#root {
+  margin: 0;
+}
+.album-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  width: 100%;
+  justify-content: center;
+  gap: 2rem;
+  margin: auto;
+  align-items: center;
+}
+
+.album {
+  flex-direction: column;
+  max-width: 300px;
+}
+.artist-name a {
+  font-size: 14px;
+  color: #a0a0a0;
+  text-decoration: transparent;
```

Comment:
I like this way of hiding the underline rather than removing it and adding it back in again for the hover